scripts/semconv/.gitignore

```diff
@@ -1 +1,2 @@
 opentelemetry-specification/
+semantic-conventions/
```

Comment:
Added a new ignore. Kept the old one since a lot of people have already checked it out and it isn't hurting anything